settings.gradle

```diff
@@ -42,6 +42,7 @@ include 'nat'
 include 'besu'
 include 'plugin-api'
 include 'plugins:rocksdb'
+include 'plugins:localfile-securitymodule'
```

Comment:
Is there a reason to break alphabetical ordering?
('plugins:localfile-securitymodule' would be above 'plugins:rocksdb')

Reply:
oversight, fixed.